utils/build/declarations.d.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+/* Globals */
+declare const PACKAGE_VERSION: string;
+
+/* Testing mocks */
+declare module '@zendeskgarden/react-testing';
```

Comment:
fwiw, we shouldn't have a dependency from `react-containers` into `react-components`. The `getExports` function should probably just be copied into this repo's "utils" folder. In general, it probably won't be used/needed in the `next` branch of `react-components`.

Reply:
Agreed. We don't currently have a global mock for `garden-test-utils` in this repo yet so this would be a larger change that should probably happen in a PR. Would be happy to open that separately.